src/lib/components/scheduling/SongIntersectionCell.svelte

```diff
@@ -22,25 +33,27 @@
 	export let song1: SongWithMusicians
 	export let song2: SongWithMusicians
 
-	let conflictingParticipants = calculateMusicianIntersection(song1, song2)
+	let songIntersection = calculateMusicianIntersection(song1, song2)
 
 	let colorClass =
 		rowIndex === columnIndex
 			? 'black-cell'
-			: conflictingParticipants.length > 0
+			: songIntersection.conflictingMusicianParticipants.length > 0
 			? 'red-cell'
+			: songIntersection.conflictingBandcoachParticipants.length > 0
+			? 'orange-cell'
 			: 'green-cell'
 
-	// When the two songs are the same or the intersection between the songs is empty, no tooltip is needed
-	let withTooltip = rowIndex !== columnIndex && conflictingParticipants.length > 0
+	// When the two songs are the same or there is no conflict between the songs, no tooltip is needed
+	let withTooltip = rowIndex !== columnIndex && songIntersection.hasConflict
```

Comment:
seperate `rowIndex === columnIndex` into a properly named const and reuse

Reply:
Check